docs/device_registry_index.md

```diff
@@ -15,11 +15,12 @@ The device registry is a registry where Home Assistant keeps track of devices. A
 | id | Unique ID of device (generated by Home Assistant)
 | name | Name of this device
 | connections | A set of tuples of `(connection_type, connection identifier)`. Connection types are defined in the device registry module.
-| identifiers | Identifiers identify the device in the outside world. An example is a serial number.
+| identifiers | Set of identifiers. They identify the device in the outside world. An example is a serial number.
 | manufacturer | The manufacturer of the device.
 | model | The model of the device.
 | config_entries | Config entries that are linked to this device.
 | sw_version | The firmware version of the device.
+| via_hub | A single identifier of a device that routes messages between this device and Home Assistant. Examples of such devices are hubs.
```

Comment:
This is hard to understand. Maybe add why this is needed, ie how this will be used by home assistant?

Reply:
Good point, update incoming. It's going to be used for our device topology.